.travis.yml

```diff
@@ -9,7 +9,7 @@ matrix:
     env:
       - TESTSPECS='src/verifications/*.spec.ts src/profile/*.spec.ts src/*.spec.ts src/votings/*.spec.ts'
       - CHAIN_ENDPOINT='ws://localhost:7545'
-      - COVERALLS_PARALLEL=true
+      - CODECOV_TOKEN="b4aa706f-0ecb-4fbc-955d-711cfea36cdb"
```

Comment:
Shouldn't we try to put access tokens into .env variables?

Reply:
its already an env var. i will put that in a travis secret :)